tests/test_ext_graphviz.py

```diff
@@ -43,3 +46,15 @@ def test_graphviz_latex(app, status, warning):
 
     macro = 'Hello \\\\includegraphics{graphviz-\w+.pdf} graphviz world'
     assert re.search(macro, content, re.S)
+
+
+@with_app('html', testroot='ext-graphviz', confoverrides={'figure_language': 'xx'})
+def test_graphviz_i18n(app, status, warning):
+    app.builder.build_all()
+    if "dot command 'dot' cannot be run" in warning.getvalue():
```

Comment:
Seeing this is now the third occurrence of `if "dot command...` in this file, could you refactor this?